shared/chat/manage-channels/edit-channel-container.js

```diff
@@ -53,7 +53,7 @@ const mergeProps = (stateProps, dispatchProps, {routeState}) => {
     channelName: stateProps.channelName,
     topic: stateProps.topic,
     onCancel: dispatchProps.onCancel,
-    onConfirmedDelete: () => dispatchProps._onConfirmedDelete(stateProps.teamname),
+    onConfirmedDelete: () => dispatchProps.onConfirmedDelete,
```

Comment:
`onConfirmedDelete: dispatchProps.onConfirmedDelete` (it isn't being called currently)

Reply:
D'oh! Fixed.